cookietemple/create/templates/gui/gui_java/{{ cookiecutter.project_slug }}/pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+g b<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
?
This must be a mistake, right?

Reply:
Yeah it is ;) wondered why the WF fails haha